assets/scripts/menus/MenuBar.jsx

```diff
@@ -86,6 +88,40 @@ class MenuBar extends React.PureComponent {
     }}))
   }
 
+  onClickSignIn = (event) => {
+    const auth0 = Authenticate()
+    auth0.authorize({
+      responseType: 'code',
+      connection: 'twitter',
+      redirectUri: AUTH0_CALLBACK_URL
+    })
+  }
```

Comment:
Is this the same as `goSignIn()` from `routing.js`?

Reply:
Yeah. It is

Reply:
Could we import `goSignIn` instead to consolidate this?

Reply:
Yeah